controlplane/src/bin/version-migration.ts

```diff
@@ -0,0 +1,90 @@
+import * as process from 'node:process';
+import { and, eq, desc, isNotNull } from 'drizzle-orm';
+import { drizzle } from 'drizzle-orm/postgres-js';
+import postgres from 'postgres';
+
+import * as schema from '../db/schema.js';
+import {
+  federatedGraphs,
+  graphCompositions,
+  schemaVersion,
+  subgraphsToFederatedGraph,
+  targets,
+  subgraphs,
+  graphCompositionSubgraphs,
+} from '../db/schema.js';
+
+const databaseConnectionUrl = process.env.DB_URL || 'postgresql://postgres:changeme@localhost:5432/controlplane';
+
+try {
+  const queryConnection = postgres(databaseConnectionUrl);
+  const db = drizzle(queryConnection, { schema: { ...schema } });
+
+  const federatedSchemaVersions = await db
```

Comment:
Please run everything in a transaction or make it idempotent and we should log a bit :smile:

Reply:
Done